.azure-pipelines/run-test-elastictest-template.yml

```diff
@@ -191,6 +205,20 @@ steps:
         pip install PyYAML
 
         rm -f new_test_plan_id.txt
+        ptf_image_type="py3only"
+        for old_branch in ${{ parameters.MIXED_PTF_IMAGE_BRANCHES }}
+        do
+          if [ $old_branch == "${{ parameters.MGMT_BRANCH }}" ]
+          then
+            ptf_image_type="mixed"
+            break
+          fi
+        done
+        PTF_IMAGETAG="--ptf_imagetag=py3only"
+        if [ $ptf_image_type == "mixed" ]
+        then
+          PTF_IMAGETAG="--ptf_imagetag=internal"
+        fi
```

Comment:
`ptf_image_type` is only used to determine whether should modify `PTF_IMAGETAG`, how about not use this extra variable,but just modify `PTF_IMAGETAG` in the for-loop statement when matching `if` condition?

Reply:
Thanks. I've modified it.